.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           - settings_reset
           - quefrency_left
           - quefrency_right
+          - nibble
```

Comment:
In addition to adding there here, you should add to `docs/static/setup.sh` and `docs/static/setup.ps1` please.

Reply:
Will do.